Tests/Engine/FactorFileTests.cs

```diff
@@ -28,7 +28,6 @@ public class FactorFileTests
         [TestFixtureSetUp]
         public void Setup()
         {
-            Config.Set("data-folder", "./TestData");
             Globals.Reset();
         }
```

Comment:
We can safely remove the entire method now, the `Globals.Reset` call was only needed because of the previous line.